docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -0,0 +1,17 @@
+`xpack.infra.enabled`:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to `true`.
+
+`xpack.infra.sources.default.logAlias`:: Index pattern for matching indices that contain log data. Defaults to `filebeat-*`.
+
+`xpack.infra.sources.default.metricAlias`:: Index pattern for matching indices that contain Metricbeat data. Defaults to `metricbeat-*`.
+
+`xpack.infra.sources.default.fields.timestamp`:: Timestamp used to for sorting log entries. Defaults to `@timestamp`.
+
+`xpack.infra.sources.default.fields.message`:: Fields used for the message for Logs UI. Defaults to `['message', '@message']`.
```

Comment:
See the comment I made previously: https://github.com/elastic/kibana/pull/26579#discussion_r238801103

I still think this description is not worded clearly.

Reply:
Sure, but I'm not sure exactly how these fields are used.

Do you mean, "Fields used to display messages in the Logs UI"?